op-chain-ops/ether/migrate.go

```diff
@@ -120,6 +121,7 @@ func doMigration(mutableDB *state.StateDB, dbFactory util.DBFactory, addresses [
 			// why we may have to filter out duplicates.
 			if seenAccounts[account.address] {
 				log.Info("skipping duplicate account during iteration", "addr", account.address)
```

Comment:
I wonder how noisey this might be if we think we might be skipping many thousand accounts?
I'd suggest getting the CSVs from Mark and testing this.